geographic_msgs/GeoPoint[] lla_points
---
geometry_msgs/Point[] map_points
